tools/test-generator.py

```diff
@@ -46,6 +47,8 @@ def send(conn, lat, lon, course, speed, alarm, ignition, accuracy, rpm, fuel):
         params = params + (('rpm', rpm),)
     if fuel:
         params = params + (('fuel', fuel),)
+    if rfid:
+        params = params + (('rfid', rfid),)
```

Comment:
Probably this script should be updated as well. By the way, have you tested everything after making changes?